src/privatesend-client.cpp

```diff
@@ -1343,9 +1344,9 @@ bool CPrivateSendClient::CreateDenominated(const CompactTallyItem& tallyItem, bo
 
 void CPrivateSendClient::RelayIn(const CDarkSendEntry& entry)
 {
-    if(!pSubmittedToMasternode) return;
+    if(!infoMixingMasternode.fInfoValid) return;
 
-    CNode* pnode = FindNode(pSubmittedToMasternode->addr);
+    CNode* pnode = FindNode(infoMixingMasternode.addr);
```

Comment:
I think this should lock cs_vNodes (like other calls to FindNode in this file do).